### ISSUE TITLE
Please optimize the connection speed of "Code Search" ❗

### ISSUE THREAD

**Issue by dnknn** [JasonBarnabe/greasyfork#761]
### https://greasyfork.org/en/search
![208_07032630](https://user-images.githubusercontent.com/10995302/89573847-d0829b80-d85d-11ea-822e-5e3d3f32c8e6.png)  `Code Search`
### Then, it is very very difficult to open, and very very slow 🐢🐌, takes at least 1 minute.

![Code Search](https://user-images.githubusercontent.com/10995302/89573679-8d282d00-d85d-11ea-9f03-64980c4a4b5d.gif)

e.g. https://greasyfork.org/en/scripts/code-search?c=translate.google.com

---
## See for details at #762

**Comment by JasonBarnabe** [comment]
It's not really meant for finding particular scripts for a site. More for finding copies of existing scripts, malware snippets, etc.

It would be nice if it's faster, though. Not sure how that would be accomplished.

**Comment by jixunmoe** [comment]
Throw everything to elastic search and expect it to work maybe?

**Comment by JasonBarnabe** [comment]
It should just take a couple of seconds now.

The solution I made was to write the latest version of every script to a directory then use grep on it. For whatever reason, this is way faster than searching in the DB.

I had also tried with a non-compressed table that contained only the latest version of every script. That took 10 seconds, which was an improvement, but this grep solution is even faster.

Only drawback is that it's not live - the index will get updated once an hour.
